src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -31,6 +31,7 @@ public partial class ScrollContentPresenter : ContentPresenter, IScrollContentPr
 		private bool _eventsRegistered;
```

Comment:
Are additional tests needed for these changes?

Reply:
I added `IsManualTest` and added instructions in an existing sample. An automated test with the current runtime test engine is likely not possible. It could be possible with UI test though.

Reply:
Thank you @Youssef1313 for adding the manual test, that would be helpful since now Darsh is helping to do manual testing for theses tests